# Determination - Deterministic rendering environment for white-axe's music
# Copyright (C) 2024 Liu Hao <whiteaxe@tuta.io>
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.

# This patch disables floating-point optimizations so that Calf's output doesn't
# change depending on what CPU the program is running on or what compiler
# version we're using.

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -12,7 +12,7 @@ pkglib_LTLIBRARIES = calf.la
 
 AM_CPPFLAGS = -I$(top_srcdir) -I$(srcdir)
 # TODO: Remove -finline flags is clang is used
-AM_CXXFLAGS = -ffast-math -finline-limit=80 $(FLUIDSYNTH_DEPS_CFLAGS) $(LV2_DEPS_CFLAGS)
+AM_CXXFLAGS = -finline-limit=80 $(FLUIDSYNTH_DEPS_CFLAGS) $(LV2_DEPS_CFLAGS)
 
 if USE_GUI
 AM_CXXFLAGS += $(GUI_DEPS_CFLAGS)
